lib/util/exec/index.ts

```diff
@@ -97,6 +97,7 @@ async function prepareRawExec(
     const extraEnv = {
       ...opts.extraEnv,
       ...customEnvVariables,
+      ...opts.userConfiguredEnv,
```

Comment:
does it work with `undefined`? 🤔

Reply:
`env` has default value `{}` so it won't be `undefined`